RELEASE-NOTES.txt

```diff
@@ -1,3 +1,7 @@
+22.6
+----
+* [**] Check required WordPress version to set galleryWithImageBlocks flag [#20736]
```

Comment:
As mentioned here, we should add the `[internal]` prefix here.

Reply:
Nitpick:
```suggestion
* [**] Check required WordPress version to set "galleryWithImageBlocks" flag [#20736]
```

Reply:
I made both changes. 👍 Out of curiosity, and so I can follow convention in the future, was the quotation marks suggestion simply to note that `galleryWithImageBlocks` is code, or is there perhaps another purpose of the quotation marks?